atc/atccmd/command.go

```diff
@@ -1407,7 +1407,7 @@ func (cmd *RunCommand) configureComponentIntervals(componentFactory db.Component
 				Interval: cmd.BuildTrackerInterval,
 			}, {
 				Name:     atc.ComponentScheduler,
-				Interval: 10 * time.Second,
+				Interval: cmd.RunnerInterval,
```

Comment:
Can you revert this one? This is a different interval.

Reply:
Sure, but is it worth creating another param? If so what would you like it to be called?